includes/admin/settings/register-settings.php

```diff
@@ -647,6 +649,8 @@ function edd_get_registered_settings() {
 							'discover'        => 'Discover',
 							'paypal'          => 'PayPal'
 						) ),
+						'tooltip_title' => __( 'Adjust the Order', 'easy-digital-downloads' ),
+						'tooltip_desc'  => __( 'Drag and drop the payment method icons to determine the order they appear in the checkout.', 'easy-digital-downloads' ),
```

Comment:
This tooltip seems redundant, as we have it above as well on the `gateways` setting as well.